### ISSUE TITLE
eli5 installation crashing on python 3.7

### ISSUE THREAD

**Issue by GuillemGSubies** [TeamHG-Memex/eli5#326]
I get `AttributeError: type object 'Callable' has no attribute '_abc_registry'` like in #273 

I am using eli5 0.9.0

It looks like this is because the `typing` dependence: https://github.com/python/typing/issues/573#issuecomment-405986724

I could send a PR if you agree to try what they propose above

**Comment by kmike** [comment]
Installing typing only for old Pythons makes sense 👍
